molo/surveys/models.py

```diff
@@ -82,8 +82,11 @@ class Meta:
 
     @property
     def clean_name(self):
-        return str(slugify(text_type(unidecode(
```

Comment:
Can't we remove this whole method? Since you added it in your original PR and the base class does it like this?